Rakefile

```diff
@@ -4,4 +4,7 @@ Yast::Tasks.configuration do |conf|
   # lets ignore license check for now
   conf.skip_license_check << /.*/
   conf.install_locations["doc/autodocs"] = conf.install_doc_dir
+
+  conf.obs_project = "home:cwh:branches:YaST:Head"
```

Comment:
NTH: This is just a temporary OBS project isn't it? Then if would be nice to have a `FIXME` comment there...